wg-serving/README.md

```diff
@@ -51,12 +51,6 @@ The WG Serving will operate in several workstreams with different areas of focus
 - [Orchestration](https://docs.google.com/document/d/1hbEx3ZEqdXCqWH9RL3uy9FIy35B8pFJ5KiK3HsOz2FE/edit?usp=sharing)
 - [DRA](https://github.com/kubernetes/community/tree/master/wg-device-management)
 
-## Sponsored Subprojects
-
-* [LLM Instance Gateway](https://github.com/kubernetes-sigs/llm-instance-gateway)
-* [Serving Catalog](https://github.com/kubernetes-sigs/wg-serving/tree/main/serving-catalog)
```

Comment:
This is not a subproject, it is at most some experimental code.

If it were a subproject, it would need to be sponsored through the relevant SIG (Probably Apps?), who already has a charter encompassing this and has the discretion over project lifecycle.

Please do not advertise subprojects without going through the processes we have in place for managing projects and avoiding conflicting ownership.

Reply:
I think we can revisit a way to link WGs to subprojects they may be participating in, but currently that's supposed to be documented by way of the SIGs they are involved with / sponsored by, and these are pretty clearly within the scope of SIG Net and SIG Apps.

I have previously requested this correction in https://github.com/kubernetes-sigs/llm-instance-gateway/issues/41, that repo also needs updating to clarify still, as it has similarly inaccurate language in the README.

To be clear: I hope you all succeed in your efforts, but we have the governance for a reason, to avoid arbitrarily hosting projects under the kubernetes umbrella and to avoid conflicts between overlapping groups. 

Work Groups are easy to create because they allow creating a place to focus on something across or within the SIGs without hammering out an ironclad, conflict-free charter. But that also means that the projects they spawn roll up to one of the SIGs.

Reply:
Thanks for the clarification!